Vagrantfile

```diff
@@ -79,7 +79,7 @@ Vagrant.configure('2') do |config|
   config.vm.define :chef do |cfg|
     config.omnibus.chef_version = nil
 
-    cfg.vm.provision :shell, inline: 'rpm -q chefdk || curl -L https://chef.sh | bash -s -- -P chefdk'
+    cfg.vm.provision :shell, inline: 'curl https://omnitruck.chef.io/install.sh | bash -s -- -c current -P chefdk'
```

Comment:
Is there a reason to drop the rpm call? This seems to try and install every time you provision now.